packages/components/scripts/jobs/vue/createWrappers.js

```diff
@@ -306,6 +311,10 @@ defineEmits<{
     ${slots}
   </${component.tagName}>
 </template>
+
+<script lang="ts">
+  ${eventExports}
+</script>
```

Comment:
When using SFCs, it seems that the agreed practice is something like:

```typescript
<script setup />
<script />
<template />
<style />
```

Don´t know if we enforce this currently.